Automattic/simplenote-macos
# Simperium: Sync'ing Account Entity

### Details
- Updates Simperium to 1.4
- Enables Sync'ing of the Account bucket, backed by **SPJSONStorage**
- Implements the **EmailVerification** entity

@eshurakov May I trouble you with this one?
Thanks in advance!

Ref #800

---

### Test: Unverified Account
0. **Update your mobile-secrets first please!**
1. Checkout commit 181efb6e13a22597c7854872d01cbaec37fb28ad (contains debug code!)
2. Log into an unverified account

- [x] Verify that the legend `Email Verification` is never printed in your console

### Test: Verified Account
0. **Update your mobile-secrets first please!**
1. Checkout commit 181efb6e13a22597c7854872d01cbaec37fb28ad (contains debug code!)
2. Log into an verified account

- [x] Verify that the legend `Email Verification` does get printed in your console

### Test: Logout
1. Log into an account that has the `email-verification` entity
2. Logout

- [x] Verify the `email-verification` entity is nuked after logout

### Release
These changes do not require release notes.
